src/patternfly/components/Table/table.scss

```diff
@@ -390,10 +391,22 @@
 .pf-c-table__sort {
   // override pf-m-plain color
   .pf-c-button {
+    // stylelint-disable
     &.pf-m-plain {
       color: var(--pf-c-table__sort--c-button--Color);
     }
   }
+  // stylelint-enable
+
+  &.pf-m-wrap {
+    .pf-c-button {
+      display: flex;
+    }
+
+    .pf-c-table__sort--c-button-text {
+      white-space: normal;
```

Comment:
did you want to move this out and apply it to `.pf-c-table__sort.pf-m-wrap .pf-c-button` above? That will fix the bug for users now, rather than relying on them updating their structure to get this.

Reply:
oh I have it moved out this might not have the latest changes